packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -28,7 +28,12 @@ class WebApplicationPackage extends ApplicationPackage {
 }
 
 class WebDevice extends Device {
-  WebDevice() : super('web', category: Category.web);
+  WebDevice() : super(
+      'web',
+      category: Category.web,
+      platformType: PlatformType.web,
+      ephemeral: true,
```

Comment:
I'm not familiar with how this one works, but I would've imagined it'd behave like desktop (eg. always be available in the device list, not appear/disappear? if so, shouldn't this be false?)